Olha_Lesiuk/HW_8/task_4.py

```diff
@@ -0,0 +1,46 @@
+def maximum(x: list, amount_of_result=1):
+    """
+            Implementing my own max function.
+            """
+
+    max_number = x[0]
+    second_max_number = x[0]
+    if amount_of_result >= 2 or amount_of_result < 1:
+        raise Exception("The function should return 2 max values from the list")
```

Comment:
why are only 2 numbers allowed? 2 in the task was as example not limitation.

Reply:
Changed logic